main.js

```diff
@@ -31,9 +31,10 @@ var mantras = [
   `Onward and upward.`,
   `I am the sky, the rest is weather.`
 ];
-// var newAffirmations = [];
-// var newMantras = [];
+var currentMessage = '';
+
 //--------------------/querySelectors/----------------------//
+
 var affirmationSelect = document.querySelector('#show-affirmation');
```

Comment:
Look into using document.getElementById() instead of querySelector. Try to query everything in the main.js by id and only use classes for css

Reply:
Interesting! I see where that would help keep things separate and more easily searchable. The teachers asked us not to use many id's this go around for reasons I'm not too sure of, but I see why it would be beneficial.